source/OutfitInfoDisplay.cpp

```diff
@@ -204,7 +211,14 @@ namespace {
 		{"hyperdrive", "Allows you to make hyperjumps."},
 		{"jump drive", "Lets you jump to any nearby system."},
 		{"minable", "This item is mined from asteroids."},
-		{"atrocity", "This outfit is considered an atrocity."}
+		{"atrocity", "This outfit is considered an atrocity."},
+		{"cloaked action", "With this cloak you can do anything as normal when cloaked, except for firing."},
```

Comment:
I don't see a need for this "can do anything" attribute when we have all the specific attributes already.

The rest of the keywords look good to me.

Reply:
Maybe it's a catch all for any future types of actions we might introduce or have missed?

Reply:
it includes them in one keyword for ease of use by CC, I think its fine?

Reply:
Since its the only thing slowing down the PR I removed it, we can always do it in another one